tests/NHSD.BuyingCatalogue.Testing.Data/Entities/CatalogueItemEntity.cs

```diff
@@ -19,24 +19,27 @@ public sealed class CatalogueItemEntity : EntityBase
         public string SupplierId { get; set; }
 
         protected override string InsertSql => @"
-            INSERT INTO dbo.CatalogueItem
-            (
-                CatalogueItemId,
-                [Name],
-                Created,
-                CatalogueItemTypeId,
-                SupplierId,
-                PublishedStatusId
-            )
-            VALUES
-            (
-                @CatalogueItemId,
-                @Name,
-                @Created,
-                @CatalogueItemTypeId,
-                @SupplierId,
-                @PublishedStatusId
-            );";
+        IF NOT EXISTS(SELECT * FROM dbo.CatalogueItem WHERE CatalogueItemId=@CatalogueItemId)
+            BEGIN
+                INSERT INTO dbo.CatalogueItem
+                (
+                    CatalogueItemId,
+                    [Name],
+                    Created,
+                    CatalogueItemTypeId,
+                    SupplierId,
+                    PublishedStatusId
+                )
+                VALUES
+                (
+                    @CatalogueItemId,
+                    @Name,
+                    @Created,
+                    @CatalogueItemTypeId,
+                    @SupplierId,
+                    @PublishedStatusId
+                );
+            END";
```

Comment:
I don't think we should need to change this, as this should be handled by the explicit call to insert a catalogue item entity that was commented out (see my earlier comment).

Reply:
Query reverted to previous version(without IF EXIST)